sandbox/pc2sandbox.sh

```diff
@@ -212,20 +232,26 @@ then
 else
 	# Get cpu time
 	cputime=`grep usage_usec $PC2_SANDBOX_CGROUP_PATH/cpu.stat | cut -d ' ' -f 2`
+	# Get wall time - we want it as close as possible to when we fetch the cpu time so they stay close
+	# since the cpu.stat includes the time this script takes AFTER the submission finishes.
+	endtime=`GetTimeInMicros`
 	if test "$cputime" -gt "$TIMELIMIT_US"
 	then
-		DEBUG echo The command was killed due to out of CPU Time "($cputime > $TIMELIMIT_US)"
+		DEBUG echo The command was killed because it exceeded the CPU Time limit "(${cputime}us > ${TIMELIMIT_US}us)"
 		COMMAND_EXIT_CODE=${FAIL_TIME_LIMIT_EXCEEDED}
 	elif test "$COMMAND_EXIT_CODE" -ge 128
 	then
 		DEBUG echo The command terminated abnormally with exit code $COMMAND_EXIT_CODE
+	else
+		walltime=$((endtime-starttime))
+		DEBUG echo The command terminated normally and took ${cputime}us / ${TIMELIMIT_US}us and ${walltime}us wall time
```

Comment:
I appreciate that there's now a "default else output" added here.  However, I think if I was a user reading the above output I'd be confused.  I believe I would see something like:
```
The command terminated normally and took 1768642us / 2000000us and 2786422us wall time
```
The expression `1768642us / 2000000u` looks to me like it's supposed to be a fraction.  (In fact, it is; it's the fractional portion of the allowed time limit that the process actually took -- but I think a user might have trouble understanding that. )  I would rather see it say something like
```
The command terminated normally and took 1768642us of CPU time (out of the CPU limit of 2000000us) and took 2786422us wall time
```
... or something similar.

Reply:
Change in [this commit](https://github.com/pc2ccs/pc2v9/pull/732/commits/920b89f78b5ead96aebb4c8bee867850fe763542).  Message now reads:

`The command terminated normally and took 4166us of CPU time (out of the CPU limit of 2000000us) and 4807us wall time`